multiversx_sdk_network_providers/api_network_provider.py

```diff
@@ -116,8 +116,8 @@ def query_contract(self, query: IContractQuery) -> ContractQueryResponse:
 
         return ContractQueryResponse.from_http_response(response)
 
-    def get_transaction(self, tx_hash: str) -> TransactionOnNetwork:
-        response = self.do_get_generic(f'transactions/{tx_hash}')
+    def get_transaction(self, tx_hash: str, with_process_status: Optional[bool] = False) -> TransactionOnNetwork:
+        response = self.do_get_generic(f'transactions/{tx_hash}?withProcessStatus={with_process_status}')
```

Comment:
I think this function should stay unmodified for the `ApiNetworkProvider` (since its completion status was good anyway). Also, I think there is no `withProcessStatus=...` on the API (and there should not be).

Reply:
This is now same as before